factory/22.04/stretch_ros2_humble.repos

```diff
@@ -6,11 +6,11 @@ repositories:
   moveit_visual_tools:
     type: git
     url: https://github.com/ros-planning/moveit_visual_tools
-    version: 4.0.0
+    version: 4.1.0
   moveit_task_constructor:
```

Comment:
@hello-atharva Binaries for this package are now available. Do we need to source build?

Reply:
Looks like it! Should we install it here or add that package as a rosdep dependency in stretch_ros2?

Reply:
The latter sounds like a better idea. Could you add it?